README.md

```diff
@@ -12,164 +12,41 @@ pip install shinywidgets
 
 ## Overview
 
-Every Shiny app has two main parts: the user interface (UI) and server logic.
-`{shinywidgets}` provides `output_widget()` for defining where to place a widget in the UI
-and `register_widget()` (or `@render_widget`) for supplying a widget-like object to
-the `output_widget()` container. More technically, widget-like means:
-
-* Any object that subclasses `{ipywidgets}`'s `Widget` class.
-* Some other widget-like object that can be coerced into a `Widget`. Currently, we
-  support objects from `{altair}`, `{bokeh}`, and `{pydeck}`, but [please let us
-  know](https://github.com/rstudio/py-shinywidgets/issues/new) about other packages that we
-  should support.
-
-The recommended way to incorporate `{shinywidgets}` widgets into Shiny apps is to:
-
-1. Initialize and `register_widget()` _once_ for each widget.
-    * In most cases, initialization should happen when the user session starts (i.e.,
-      the `server` function first executes), but if the widget is slow to initialize and
-      doesn't need to be shown right away, you may want to delay that initialization
-      until it's needed.
-2. Use Shiny's `@reactive.Effect` to reactively modify the widget whenever relevant
-   reactive values change.
-3. Use `{shinywidgets}`'s `reactive_read()` to update other outputs whenever the widget changes.
-    * This way, relevant output(s) invalidate (i.e., recalculate) whenever the relevant
-      widget attributes change (client-side or server-side).
-
-The following app below uses `{ipyleaflet}` to demonstrate all these concepts:
+See the [using ipywidgets section](https://shiny.rstudio.com/py/docs/ipywidgets.html) of the Shiny for Python website.
```

Comment:
Don't merge until it's clear https://github.com/rstudio/pyshiny-site/pull/64 will get merged